OpenRA.Mods.Common/Traits/GivesExperience.cs

```diff
@@ -21,6 +21,9 @@ class GivesExperienceInfo : ITraitInfo
 		[Desc("Grant experience for team-kills.")]
 		public readonly bool FriendlyFire = false;
 
+		[Desc("Grant player experience for kills.")]
+		public readonly bool GivesPlayerExperience = false;
```

Comment:
My impression was non combat objects outnumber combat objects, hence making it false by default.